apis/nucleus/src/object/navigation/navigation.js

```diff
@@ -10,10 +10,37 @@ export default function createNavigationApi(halo, store, model) {
 
   /**
    * @class Navigation
+   * @description The navigation api instance.
+   * @experimental
+   * @since 5.4.0
+   * @example
+   * const navigation = useNavigation();
+   * //...
+   * useEffect(() => {
+   *   const onSheetChanged = () => {
+   *     setActiveSheetId(navigation?.getCurrentSheetId() || "");
```

Comment:
Replace this line with a `...` just so we don't reference anything that isn't explained.